pysat/instruments/icon_mighti.py

```diff
@@ -0,0 +1,252 @@
+# -*- coding: utf-8 -*-
+"""Supports the Michelson Interferometer for Global High-resolution 
+Thermospheric Imaging (MIGHTI) instrument onboard the Ionospheric
+CONnection Explorer (ICON) satellite.  Accesses local data in
+netCDF format.
+
+Parameters
+----------
+platform : string
+    'icon'
+name : string
+    'mighti'
+tag : string
+    'level_2'
+sat_id : string
+    'Red' or 'Green'
+
+Warnings
+--------
+- The cleaning parameters for the instrument are still under development.
+- Only supports level-2 data.
+
+Example
+-------
+    import pysat
+    mighti = pysat.Instrument('icon', 'mighti', clean_level='clean')
+    mighti.download(pysat.datetime(2019, 1, 30), pysat.datetime(2019, 12, 31))
+    mighti.load(2017,363)
+
+Authors
+---------
+Originated from EUV support.
+Jeff Klenzing, Mar 17, 2018, Goddard Space Flight Center
+Russell Stoneback, Mar 23, 2018, University of Texas at Dallas
+Conversion to MIGHTI, Oct 8th, 2028, University of Texas at Dallas
+
+"""
+
+from __future__ import print_function
+from __future__ import absolute_import
+
+import functools
+
+import pandas as pds
+import numpy as np
+
+import pysat
+
+
+platform = 'icon'
+name = 'mighti'
+tags = {'level_2':'Level 2 public geophysical data'}
+sat_ids = {'Green':['level_2 Green Line'],
```

Comment:
Shouldn't the sat_ids be all lower case?

Reply:
They should.